admin/lint_json.m

```diff
@@ -0,0 +1,52 @@
+function lint_json(filesin, outputfile)
```

Comment:
This would benefit from a header comment as it's not clear what it's doing from the name